model/src/pyrenew/process/deterministic.py

```diff
@@ -0,0 +1,73 @@
+from pyrenew.distutil import validate_discrete_dist_vector
+from pyrenew.metaclass import RandomVariable
+
+
+class DeterministicProcess(RandomVariable):
```

Comment:
Noting that we plan to discuss a bit more whether and if so how to use this construct.

Reply:
Another thing to consider if we do: the name. `DeterministicProcess` or `DeterministicVariable`?

Reply:
Since it is under process, I voted for process :P. One thing worth discussing is that both process/deterministic.py and observation/deterministic.py are 99% the same, only difference is the name, so it makes me wonder if it should be moved to pyrenew under metaclass or something.